content/rs/release-notes/rs-5-4-6-july-2019.md

```diff
@@ -0,0 +1,85 @@
+---
+Title: Redis Enterprise Software Release Notes 5.4.6 (July 2019)
+description: 
+weight: 84
+alwaysopen: false
+categories: ["RS"]
+---
+
+[Redis Enterprise Software (RS) 5.4.6](https://redislabs.com/redis-enterprise/software/downloads/#downloads) is now available.
+This release includes the latest version of [Redis 5 (5.0.5](https://redis.io/)), bundles the GA release of the new RedisTimeSeries module,
+and adds other enhancements & bug fixes.
+
+## Overview
+
+Follow these [instructions]({{< relref "/rs/installing-upgrading/upgrading.md" >}}) for upgrading to RS 5.4.6 from RS 5.0 and above.
+If you have a version older than 5.0, you must first upgrade to version 5.2 (or at least 5.0).
+
+## New Features
+
+### Time Series
+
+New GA release of the [RedisTimeSeries](https://redislabs.com/blog/redistimeseries-ga-making-4th-dimension-truly-immersive/) (version v1.0.0) module is bundled in RS 5.4.6.
+
+### Modules Versions
+
+Updated versions of GA modules:
+
+- [RedisTimeSeries](https://oss.redislabs.com/redistimeseries/)  - v1.0.0 (new module)
+- [RedisBloom](https://oss.redislabs.com/redisbloom/)   -  v2.0.0 (version update)
+- [RediSearch](https://oss.redislabs.com/redisearch/)  -  v1.4.11 (version update)
+- [RedisGraph](https://oss.redislabs.com/redisgraph/)  -  v1.2.2 (no change)
+- [RedisJSON](https://oss.redislabs.com/redisjson/)  -  v1.0.3 (no change)
+
+## Additional Capabilities
+
+- [Latest version of Redis 5](https://redis.io/) (5.0.5) was merged into RS 5.4.6.
+- If a user subscribes to a channel during recovery of a Active-Active Redis (CRDB) from an Append Only File (AOF), the user receives only the new messages.
+    (Fixes known limitation from RS 5.4.4)
+- CROSSSLOT behavior now matches Redis OSS behavior for accessing multiple keys in a single command.
+    Now, when you use CRC16 hash function (default), you can pass a command for different keys/tags that are mapped to the same slot. (RS23189)
+- Cluster upgrade process was improved so that the duration of the cluster upgrade process reduced.
+
+## Information
+
+- End of Life (EOL) for Redis Enterprise Software 5.4, as well as for Redis Modules and previous RS versions,
+    can be found [here]({{< relref "/rs/administering/product-lifecycle.md" >}}).
+
+## Important Fixes
+
+- RS28679 - More Active-Active Redis (CRDB) statistics are exposed through Prometheus.
```

Comment:
Let's add in the end:
"(Syncer and peers related statistics which are already available with the REST API)"
What do you say?

Reply:
How about: Syncer and peers related statistics for Active-Active Redis (CRDB), which are already available with the REST API, are exposed through Prometheus

Reply:
How about: Syncer and peers related statistics for Active-Active Redis (CRDB), which are already available with the REST API, are exposed through Prometheus